packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -217,7 +239,13 @@ contract L2OutputOracle is Initializable, Semver {
             );
         }
 
-        emit OutputProposed(_outputRoot, nextOutputIndex(), _l2BlockNumber, block.timestamp);
+        // Post the bond to the bond manager
+        BOND_MANAGER.post{ value: msg.value }(
+            keccak256(abi.encode(_l2BlockNumber)),
```

Comment:
nit: use kwargs syntax so its easier to see what the args are
Also why hash the blocknumber? There can be colliding ids..